pkg/kubectl/resource/visitor.go

```diff
@@ -222,7 +222,7 @@ func (v *PathVisitor) Visit(fn VisitorFunc) error {
 		if !v.IgnoreErrors {
 			return err
 		}
-		fmt.Fprintf(os.Stderr, "error: unable to load file %q: %v\n", v.Path, err)
+		fmt.Fprintf(os.Stderr, "error: %v\n", err)
```

Comment:
@mikedanese  Because InfoForData function prints errors that include the source, ValidateSchema function doesn't have the path parameter, so the errors that not include the source. ioutil.ReadFile function has the path parameter, the errors that include the source too, we should remove the duplicate source too. I think if the function has the  path parameter, the errors that include the source. if not, we should add source to errors. What do you think about these?